app/blog/2018-01-10-polymer-3-new-preview.md

```diff
@@ -0,0 +1,102 @@
+
+The latest version of Modulizer features better conversion, especially around test and demo paths. This is still an early preview for elements. Many more test suites are running and passing than in Preview 1, but there's still a lot of work to do on Modulizer to get even more to pass. Most elements work well enough for you to try them out.
+
+The 3.0 preview release of the Polymer core library includes two new features:
+
+* `PropertiesMixin`. This mixin breaks out a small subset of the lower Polymer layers responsible for defining declarative properties, creating accessors, and syncing attributes to properties. 
+
+  Users of lit-html and other templating alternatives may find `PropertiesMixin` useful. It provides a single `_propertiesChanged` callback and can be used to create new, lightweight base classes. 
+
+  To illustrate, the following example shows a very simple element using `PropertiesMixin`:
+
+  ```js
+  import {PropertiesMixin} from "../node_modules/@polymer/polymer/lib/mixins/properties-mixin.js"
+
+    class MyPropertiesElement extends PropertiesMixin(HTMLElement) {
+      // Define properties to watch. Only types are supported.
+      static get properties() {
+        return {
+          name: String
+        }
+        // Called whenever the declared properties change. 
+        _propertiesChanged(currentProps, changedProps, oldProps) {
+          // Render the changed content.
+          this.textContent = `Hello, ${this.name}`;
+        }
+      }
+    }
+  }
+  ```
+
+  For more information, see the source for `PropertiesMixin`. Documentation will be coming in a future release. 
+
+  For people interested in using lit-html, an experimental base class using `PropertiesMixin` with lit-html templating is available [somewhere](https://). 
+
+* `html` tag function. Starting with this preview, you need to return an instance of `HTMLTemplateElement` from your element’s static template getter—the option to return a string has been removed. This change makes the template type predictable, which in turn will make it simpler to extend superclass templates.
+
+  The easiest way to return an `HTMLTemplateElement` from your template getter is to use Polymer’s new `html` helper, which automatically generates an `HTMLTemplateElement` from a template literal string, as shown here: 
+
+  ```js
+  import {Element as PolymerElement, html} from "../node_modules/@polymer/polymer/polymer-element.js"
+  class MyAppElement extends PolymerElement {
+    static get template() {
+      return html`<div>I'm a template</div>
+                  <div>[[withBindings]]</div>
+                  <button on-click="clickHandler">Click me!</button>`
+    }
+    ...
+  }
+  customElements.define('my-app-element', MyAppElement);
+  ```
+
+These new features will also be released in the next minor release of 2.x. However, there's no need to manually convert your 2.x code to use the `html` helper—the Polymer Modulizer will make that conversion for you.
```

Comment:
In the interest of forward compatibility, these new features have been implemented in the Polymer 2 codebase and will also be included in the next 2.x release. As with any new features, keep in mind that they're subject to change until they land in an official release.

By the way, though you'll be able to use the `html` helper in your 2.x code if you want, you don't need to manually convert all of your string-based templates–Polymer Modulizer will do that for you when you convert to 3.0.